week1/woojin/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        table, th, td {
+            border : 1px solid black;
+            border-collapse : collapse;
+        }
+        td {
+            width: 100px;
+            text-align: center;
+        }
+    </style>
+    <title>Document</title>
+</head>
+<body>
+    <h1>안녕하세요 :) </h1>
+    <h2>저의 이름은 <strong>장우진 </strong>입니다!</h2><br>
```

Comment:
strong 태그를 사용해주셨네요! 
strong 태그는 텍스트의 의미적 중요성을 나타내는 태그입니다. b 태그와 em 태그와의 차이점을 같이 알아보시면 좋을 것 같아요! 
그리고 h2 태그는 블록(block) 요소여서 자동으로 줄바꿈이 일어나기 때문에 br 태그는 사용 안해주셔도 됩니다! 만약 두개의 h2 태그 사이에 의도적으로 공간을 두려고 쓴것이라면 css를 사용해서 margin이나 padding을 조절하는것이 더 좋은 방법입니다. 
또 여기서도 닫는 태그 앞에 공백이 있는데, 제거 해주시는게 좋습니다.
```suggestion
    <h2>저의 이름은 <strong>장우진</strong>입니다!</h2>
```